index.html

```diff
@@ -23,25 +23,152 @@ <h1>Hey there. I’m Mariane <br/><span>I’m a software developer</span></h1>
+            <div class="card">
+                <div class="card-body">
+                    <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+                    <ul>
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <a href="" class="see-more">See Project</a>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-body">
+                    <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+                    <ul>
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <a href="" class="see-more">See Project</a>
+                </div>
+            </div>
+            <div class="card">
+                <div class="card-body">
+                    <h3 class="card-title">Multi-Post Stories Gain+Glory</h3>
+                    <ul>
+                        <li>Ruby on rails</li>
+                        <li>css</li>
+                        <li>JavaScript</li>
+                        <li>html</li>
+                    </ul>
+                    <a href="" class="see-more">See Project</a>
+                </div>
+            </div>
+        </div>
+    </section>
+    <div class="aboutme">
+        <h3>About me</h3>
+        <p> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <a href="" class="get-my-resume">Get My Resume</a>
+        <ul>
+            <li class="aboutme-category">
+                <img src="/images/languages.png" alt="languages">
+                <h3>Languages</h3>
+                <ul>
+                    <li>Javascipt</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>CSS</li>
+                </ul>
+            </li>
+            <li class="aboutme-category">
+                <img src="/images/frameworks.png" alt="frameworks">
+                <h3>Frameworks</h3>
+                <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on rails</li>
+                    <li>RSpec</li>
+                    <li>Capybara</li>
+                    <li>Selenium</li>
+                </ul>
+            </li>
+            <li class="aboutme-category">
+                <img src="/images/skills.png" alt="skills">
+                <h3>Skills</h3>
+                <ul>
+                    <li>Codekit</li>
+                    <li>GitHub</li>
+                    <li>Codepen</li>
+                    <li>Gitlab</li>
+                    <li>Terminal</li>
+                </ul>
+            </li>
+        </ul>
```

Comment:
Great work on this section 🎊  but in order to make the design better and make it look more like Figma, you could consider centering this section by giving it the right margins. 🛰️  Also make sure to fix the image's link, it's always better with the images. 🖼️  See below 👇🏽 

## Your design
![Screenshot 2023-01-13 at 21 45 08](https://user-images.githubusercontent.com/45739963/212447332-59b8895a-6adf-4e09-bb09-ca4d789eb2cf.png)

## Figma Design
![Screenshot 2023-01-13 at 21 45 32](https://user-images.githubusercontent.com/45739963/212447347-ecdd5651-d7bc-4e2e-9605-e46f1f9d9cdb.png)